docset/winserver2025-ps/hyper-v/New-VM.md

```diff
@@ -120,16 +129,25 @@ Accept wildcard characters: False
 
 ### -BootDevice
 
-Specifies the device to use as the boot device for the new virtual machine.
-Allowed values are **CD**, **Floppy**, **LegacyNetworkAdapter**, **IDE**, NetworkAdapter, and VHD.
+Specifies the device to use as the boot device for the new virtual machine. Acceptable values are:
+
+- `CD`
+- `Floppy`
+- `LegacyNetworkAdapter`
+- `IDE`
+- `NetworkAdapter`
+- `VHD`
 
-When LegacyNetworkAdapter is specified, this configures the new virtual machine with a network adapter that can be used to perform a PXE boot and install an operating system from a network installation server.
+When `LegacyNetworkAdapter` is specified, this configures the new virtual machine with a network
+adapter that can be used to perform a PXE boot and install an operating system from a network
```

Comment:
This should be updated to specify a PXE boot _with a 32-bit operating_ system.  We don't want to encourage use of this flag.